go.mod

```diff
@@ -2,7 +2,10 @@ module github.com/transferwise/crypto
 
 go 1.14
 
+replace golang.org/x/crypto => github.com/ProtonMail/crypto v0.0.0-20201112134528-b4bfec6bba36
```

Comment:
this is the supplementary crypto lib, which is not the core crypto we are using for AES, DES and RSA:
https://github.com/golang/go/tree/master/src/crypto

Reply:
the main reason for this change is due to Golang module and ProtonMail crypto is already on V2